https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=239266 reports how
clang unfortunately poisons user namespace by default (without any
special options).

Until that changes (or GCC changes) we need to avoid using vec_step
as a variable.

--- UTC
Index: gcc/tree-vect-loop.cc
===================================================================
--- gcc/tree-vect-loop.cc	(revision 273856)
+++ gcc/tree-vect-loop.cc	(working copy)
@@ -55,6 +55,8 @@ along with GCC; see the file COPYING3.  If not see
 #include "vec-perm-indices.h"
 #include "tree-eh.h"
 
+#define vec_step vec_step_
+
 /* Loop Vectorization Pass.
 
    This pass tries to vectorize loops.
